pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -454,6 +454,20 @@ func TestOpenShiftClusterStaticValidateNetworkProfile(t *testing.T) {
 			},
 			wantErr: "400: InvalidNetworkAddress: properties.networkProfile.serviceCidr: The provided service CIDR '10.0.150.0/16' is invalid, expecting: '10.0.0.0/16'.",
 		},
+		{
+			name: "podCidr invalid CIDR",
+			modify: func(oc *OpenShiftCluster) {
+				oc.Properties.NetworkProfile.PodCIDR = "100.64.0.0/16"
+			},
+			wantErr: "400: InvalidCIDRRange: properties.networkProfile: OpenShift Container Platform uses 100.64.0.0/16 IP address range internally. Do not incude this 100.64.0.0/16 IP address in any other CIDR definitions in your cluster.",
+		},
+		{
+			name: "serviceCidr invalid CIDR",
+			modify: func(oc *OpenShiftCluster) {
+				oc.Properties.NetworkProfile.ServiceCIDR = "100.64.0.0/16"
+			},
+			wantErr: "400: InvalidCIDRRange: properties.networkProfile: OpenShift Container Platform uses 100.64.0.0/16 IP address range internally. Do not incude this 100.64.0.0/16 IP address in any other CIDR definitions in your cluster.",
```

Comment:
Fix typo, and use IP address range here as well.

Use Azure Red Hat OpenShift here, instead of OpenShift Container Platform